client/my-sites/theme/main.jsx

```diff
@@ -450,6 +458,29 @@ const ThemeSheet = React.createClass( {
 		return <ThemesRelatedCard currentTheme={ this.props.id } />;
 	},
 
+	renderRetired() {
+		return (
+			<div className="theme__sheet-content">
+				<Card className="theme__retired-theme-message">
+					<Gridicon icon="cross-circle" size={ 48 } />
+					<div className="theme__retired-theme-message-details">
+						<div className="theme__retired-theme-message-details-title">{ i18n.translate( 'This theme is retired' ) }</div>
+						<div>
+							{ i18n.translate( 'We invite you to try out a newer theme; start by browsing our WordPress theme directory.' ) }
+						</div>
+					</div>
+					<Button
+						primary={ true }
+						href={ '/themes/' }>
+						{ i18n.translate( 'See all themes' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **27** times:
`translate( 'See All Themes' )` *ES Score: 11*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).